README.rst

```diff
@@ -2,8 +2,8 @@ CenPy
 =====
 .. image:: https://img.shields.io/static/v1.svg?label=documentation&message=latest&color=blueviolet
     :target: https://cenpy-devs.github.io/cenpy
-.. image:: https://travis-ci.org/cenpy-devs/cenpy.svg?branch=master
-    :target: https://travis-ci.org/cenpy-devs/cenpy
+.. image:: https://github.com/cenpy-devs/cenpy/actions/workflows/build.yml/badge.svg
+    :target: https://github.com/cenpy-devs/cenpy/actions/workflows/build.yml
```

Comment:
Using the following target URL will resolve to the actual status, not simply the file:
```
https://github.com/cenpy-devs/cenpy/actions?query=workflow%3A.github%2Fworkflows%2Funittests.yml
```

An example can be seen in [`libpysal`](https://github.com/pysal/libpysal/blame/master/README.rst#L6).